modules/remotecontrol/Receiver.js

```diff
@@ -128,6 +135,7 @@ export default class Receiver extends RemoteControlParticipant {
             APP.conference.addConferenceListener(ConferenceEvents.USER_LEFT,
                 this._userLeftListener);
             this.controller = userId;
+            logger.debug("Remote control permissions granted to: " + userId);
```

Comment:
Same as the comment above, will it make sense to have it logged on the "log" level ?